docker-compose.yml

```diff
@@ -1,27 +1,32 @@
 version: "3.8"
 services:
-  server:
-    container_name: "sceta-server"
-    restart: always
-    build:
-      dockerfile: ./Dockerfile
-      context: .
-    command:
-      - --config=/server/data.yaml
-      - --host=0.0.0.0
-      - --port=8001
-    volumes:
-      - ./data.yaml:/server/data.yaml
-      - ./app.py:/server/app.py
-      - ./modules:/server/modules
-    ports:
-      - 8001:8001
-  prometheus:
-    image: prom/prometheus:latest
-    restart: always
-    volumes:
-      - ./prometheus.yml:/etc/prometheus/prometheus.yml
-    command:
-      - "--config.file=/etc/prometheus/prometheus.yml"
-    ports:
-      - 9090:9090
+    server:
+        container_name: "sceta-server"
+        restart: always
+        build:
+            dockerfile: ./Dockerfile
+            context: .
+        command:
+            - --config=/server/data.yaml
+            - --host=0.0.0.0
+            - --port=8001
+        volumes:
+            - ./data.yaml:/server/data.yaml
+            - ./app.py:/server/app.py
+            - ./modules:/server/modules
+        ports:
+            - 8001:8001
```

Comment:
we dont need this port open anymore, everything will pass through nginx, which will come from clark

like how we interact with cleezy from sce.sjsu.edu here https://github.com/SCE-Development/Clark/wiki/Our-Code-Structure

Reply:
Done